Amplify/Categories/Auth/AuthCategory+ClientBehavior.swift

```diff
@@ -113,4 +93,25 @@ extension AuthCategory: AuthCategoryBehavior {
                                      options: options,
                                      listener: listener)
     }
+
+    public func resetPassword(for username: String,
+                              options: AuthResetPasswordOperation.Request.Options? = nil,
+                              listener: AuthResetPasswordOperation.EventListener?) -> AuthResetPasswordOperation {
+        return plugin.resetPassword(for: username,
+                                    options: options,
+                                    listener: listener)
+    }
+
+    public  func confirmResetPassword(for username: String,
+                                      with newPassword: String,
```

Comment:
Since we had already gone through naming review, I defer this discussion to later to finally go through all the apis.
Created a GH task for this https://github.com/aws-amplify/amplify-ios/issues/424

Reply:
Sounds good, added label Auth and P1  --seems like this needs to get addressed before GA?